dev/drivers/scripts/plots/global_ens/jevs_global_ens_headline_gefs_grid2grid_plots.sh

```diff
@@ -7,6 +7,16 @@
 #PBS -l place=shared,select=1:ncpus=1:mem=5GB
 #PBS -l debug=true
 
+#************************************************************************
+#Note: (1) This headline plot generation driver script should be run 
+#           on Jan 18th of each year. 
```

Comment:
@BinbinZhou-NOAA Please say that this should be run on Jan 16th of each year. Congress has asks for this earlier than the Jan 18th (as they did this year). We can manually get the forecast/analysis files earlier than EVS gets it if we need if we have to. 

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA